client/packages/common/src/authentication/api/hooks/useGetRefreshToken.ts

```diff
@@ -1,30 +1,7 @@
-import { useState } from 'react';
 import { useAuthApi } from './useAuthApi';
-import { useQuery } from 'react-query';
-import { COOKIE_LIFETIME_MINUTES } from '../../AuthContext';
-import { useDebounceCallback } from '@common/hooks';
+import { useMutation } from 'react-query';
 
-export const useGetRefreshToken = (token: string) => {
+export const useGetRefreshToken = () => {
   const api = useAuthApi();
-  const [enabled, setEnabled] = useState(false);
-  const refetchInterval = Math.max(COOKIE_LIFETIME_MINUTES - 1, 1) * 60 * 1000;
-  const debouncedSetEnabled = useDebounceCallback(
```

Comment:
Note, there is a concurrency issue with useDebounceCallback (https://github.com/openmsupply/open-msupply/pull/1099) so I am always suspicious when it is/was used...